.github/workflows/test.yml

```diff
@@ -96,3 +96,33 @@ jobs:
       - name: Run tests
         run: |
           pytest -n auto
+  build-free-threading:
+    # Later we can merge this job with build similarly to
+    # https://github.com/python-pillow/Pillow/blob/f0d8fd3059bc1b291563d8a0b1f224b6fd7d0b90/.github/workflows/test.yml#L56-L57
+    name: Python 3.13 with free-threading
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
In #175 I'm updating actions to use hashes rather than tags. Can you use the same hashes within this PR?